signup.html

```diff
@@ -3,40 +3,49 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="stylesheet.css" />
+    <link rel="stylesheet" href="style.css" />
     <header>
       <div class="login">
         <h3><a href="index.html">Go Back</a></h3>
       </div>
     </header>
   </head>
   <body>
-    <div class="signup-container">
+    <div class="container">
       <form id="registrationForm">
         <h1>Registration form</h1>
-        <!--username feild -->
+        <!--username field -->
         <div class="input-control">
           <label for="name">User-name</label>
           <input id="name" name="name" type="text" />
           <span class="error" id="nameError"></span>
         </div>
-        <!--email feild -->
-
+        <!--email field -->
         <div class="input-control">
           <label for="email">User-email</label>
           <input id="email" name="email" type="email" />
           <span class="error" id="emailError"></span>
         </div>
-        <!--password feild -->
+        <!--password field -->
         <div class="input-control">
           <label for="password">Password</label>
           <input id="password" name="password" type="password" />
           <span class="error" id="passwordError"></span>
         </div>
         <button type="submit" id="submitBtn">submit</button>
         <div class="success-message" id="successMessage"></div>
+        <div id="registrationStatus" style="display: none">
+          Registration status show here
+        </div>
       </form>
     </div>
+    <footer class="linkedIn">
```

Comment:
this element is footer, not linkedin, a css classname should not be named as linkedIn. what if you want to add another `<a>` to your social e.g facebook ? would this class be "linkedInAndFacebook" ? and what if you need to add instagrtam as well? 
read about common (or best) naming conventions for variables, functions/methods, css-classnames, css-ids

Reply:
fixed!